scripts/publish-vsce.sh

```diff
@@ -28,7 +28,7 @@ echo "Making a $VERSION version of VSCode..."
 npm version $VERSION
 NEW_VSCODE_VERSION=$(jq -r ".version" package.json)
 NEXT_HEADER="## NEXT"
-NEW_HEADER="## NEXT \n\n## $NEW_VSCODE_VERSION\n\n- Updated internal firebase-tools dependency to $CLI_VERSION"
+NEW_HEADER="## NEXT\n\n## $NEW_VSCODE_VERSION\n\n- Updated internal firebase-tools dependency to $CLI_VERSION"
```

Comment:
Removing a stray space that was leaving the CHANGELOG with linter issues after each release.